include/wlr/types/wlr_idle.h

```diff
@@ -68,4 +73,15 @@ void wlr_idle_notify_activity(struct wlr_idle *idle, struct wlr_seat *seat);
  */
 void wlr_idle_set_enabled(struct wlr_idle *idle, struct wlr_seat *seat,
 	bool enabled);
+
+/**
+ * Create a new timer on the given seat. The fired event will be called after
```

Comment:
Do you mean the idle event?

Reply:
Oh yeah, I initially called this `fired` then decided `idle` is a better name